### ISSUE TITLE
Error On Mac NPM Install

### ISSUE THREAD

**Issue by stunjiturner** [lumen-cms/lumen-cms#6]
In file included from ../node_modules/nan/nan.h:190:
../node_modules/nan/nan_maybe_43_inl.h:88:15: error: no member named 'ForceSet' in 'v8::Object'
  return obj->ForceSet(GetCurrentContext(), key, value, attribs);

NAN dependency needs to be upgraded, where in the lumen-cms dependendencies is it blend called from

**Comment by dohomi** [comment]
@stunjiturner not sure whats going on in your installation. which version of Nuxt are you using? Did you use the starter template? https://github.com/lumen-cms/starter-template

**Comment by stunjiturner** [comment]
Yes I used the starter template , i even updated to NuXT 2.1, but the gql-tools .0.15 is the culprit, because when I removed it from the package.json the yarn install would complete. The version of nuxt made no difference in my investigation. From the research eslint-prettier has same issue with nan, maybe it’s called in the gql-tools dependency???

S Tunji Turner
sightuary
@sightuary

**Comment by dohomi** [comment]
this package is only compatible with the latest 1.4.2 - havent got the time to upgrade to 2.x due to apollo and graphql errors

**Comment by stunjiturner** [comment]
Will downgrade and test, later
